engines/suppliers/app/serializers/suppliers/supplier_serializer.rb

```diff
@@ -0,0 +1,8 @@
+module Suppliers
+  class SupplierSerializer < Shared::BaseSerializer
+    schema do
+      type 'supplier'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.